_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,137 @@
+tags: [snowplow, release]
+author: Łukasz Indykiewicz
+category: Releases
+permalink: /blog/2019/11/01/snowplow-r117-biskupin/
+---
+
+Snowplow release R117 is named after [Biskupin](https://en.wikipedia.org/wiki/Biskupin), which is "an archaeological site and a life-size model of a late Bronze Age fortified settlement in north-central Poland that also serves as a archaeological open-air museum".
+
+This release focuses on delivering small, but necessary improvements to Snowplow ecosystem:
+
+1. [TLS port binding and certificate](#tls-port-binding-and-certificate)
+2. [Referer parser refreshment](#referer-parser-refreshment)
+3. [IPv6 anonymization](#ipv6-anonymization)
+4. [Additional event fingerprint hashing methods (SHA1, SHA256, SHA384, SHA512)](#additional-event-fingerprint-hashing-methods)
+5. [Support to the spot market for core instances](#support-to-the-spot-market-for-core-instances)
+
+The `bad row` release has been renamed to R118 - check [roadmap](#roadmap) for more details.
+
+## TLS port binding and certificate
+
+### TLS/SSL Certificate termination
+As an additional security measure it is now possible to terminate TLS/SSL connection directly within `scala-stream-collector` using a battle-tested [lightben/ssl-config](https://lightbend.github.io/ssl-config/index.html).
+We introduce several new configuration parameters in order to fit most workflows and configurations.
+There are two configuration sections that can be overriden in order to achieve the expected workflow: `collector.ssl` and `ssl-config`.
+The former is a high-level section that allows:
+`collector.ssl.enable` - turn on ssl termination
+`collector.ssl.redirect` - whether automatic upgrade from http to https should be performed
+`collector.ssl.port` - port on which TLS/SSL server should be started
+The latter allows for low-level TLS/SSL configuration exposed by [lightbend/ssl-config](https://lightbend.github.io/ssl-config/index.html).
+​
+For example to start up an ssl-enabled, auto-upgrade server, following config can be used:
+```hocon
+ssl {
+  enable = true
+  redirect = true
+  port = 443
+}
+```
+However, this configuration will use environment-defined JVM-attached certificates. In order to override the default behaviour and use a custom certificate, the low-level section can be defined as:
+```hocon
+ssl-config {
+  keyManager = {
+    stores = [
+      {type = "PKCS12", classpath = false, path = ${CERT_FILE}, password = "pass" }
+    ]
+  }
+}
+```
+​
+### Default redirect endpoint disabling
+Another simple, but important security improvement is a possibility to disable the default event submission redirect. This allows users to disable the default endpoint in favour of custom user-defined url. 
+For example, following configuration will only allow redirects for custom-defined `/com.acme/redirect-me` endpoint, whereas the default `/r/tp2` will not be available.
+```hocon
+enableDefaultRedirect = false
+paths {
+  "/com.acme/redirect-me" = "/r/tp2"
+}
+```
```

Comment:
Does the above highlighting syntax work on our blog? I think we used to use 

{% highlight hocon %}
...
{% endhighlight %}

Not sure if the three backticks work - just checking you've tried to build the post locally to check?

Reply:
I have just double checked it, so three backticks works, but there is no support for `hocon` (backticks nor the other syntax you mentioned), so I set up it as json. It isn't perfect, but looks good enough imho.